tests/ctst/HOW_TO_WRITE_TESTS.md

```diff
@@ -0,0 +1,121 @@
+# How to write a test that runs in parallel
+
+Writting intergation / behavior tests that run in parallel is not trivial.
+This page lists all the rules to follow when writing a test that runs in
+parallel with other tests.
+
+*This is specific to Zenko, but the rules are applicable for any suite of*
+*tests in parallel.*
+
+**Rule #1**
+
+> Tests must be idempotent.
```

Comment:
```suggestion
> Tests should be idempotent.
```

Reply:
Why "should" and not "must"? it is not supposed to be a suggestion.